src/components/Select/Select.styles.js

```diff
@@ -1,14 +1,16 @@
 import styled from 'styled-components';
+import InputWrapper from '../../theme/inputs.styles';
 
-export const SelectWrapper = styled.div`
+export const SelectInputWrapper = styled(InputWrapper)`
   display: flex;
-  flex-flow: column;
-  position: relative;
+  flex-direction: column;
+
+  label {
+    font-weight: bold;
+  }
```

Comment:
The label for TextInput and TextArea is font-weight:400.  No need to include label styling as it should be in the default typography.styles

Reply:
code removed